openpower/package/machine-xml/machine-xml.mk

```diff
@@ -91,6 +91,16 @@ define MACHINE_XML_BUILD_CMDS
             $(PETITBOOT_BIOS_XML_METADATA_FILE) \
             $(PETITBOOT_XSLT_FILE) \
             $(BIOS_XML_METADATA_FILE)
+
+        # Create the wofdata
+        if [ -e $(MRW_HB_TOOLS)/wof-tables-img ]; then \
+            chmod +x $(MRW_HB_TOOLS)/wof-tables-img; \
+        fi
+
+        if [ -e $(MRW_SCRATCH)/wof_v4_dd1_export.csv ]; then \
```

Comment:
I like the generic name. Especially since everything is going through machine.xml anyway.